arenadata/Dockerfile.ubuntu

```
@@ -8,6 +13,12 @@ ARG adb_python3_bin=/opt/adb6-python3.9/bin/python3
 
 COPY README.ubuntu.bash ./
 RUN set -eux; \
+    sed -i \
+      -e "s|http://archive.ubuntu.com/ubuntu/ jammy |${UBUNTU_MAIN_MIRROR}/ jammy |g" \
+      -e "s|http://archive.ubuntu.com/ubuntu/ jammy-updates |${UBUNTU_UPDATES_MIRROR}/ jammy-updates |g" \
+      -e "s|http://archive.ubuntu.com/ubuntu/ jammy-backports |${UBUNTU_BACKPORTS_MIRROR}/ jammy-backports |g" \
+      -e "s|http://security.ubuntu.com/ubuntu/ jammy-security |${UBUNTU_SECURITY_MIRROR}/ jammy-security |g" \
+      /etc/apt/sources.list && \
```

NO!!! here `;` should be used!!!
```suggestion
      /etc/apt/sources.list; \
```